### ISSUE TITLE
Wiki API is unreliable

### ISSUE THREAD

**Issue by ayzaan** [bksim/wikisurf#7]
Seems like it's very very slow at times. If (when) we move to production, we'll definitely either have to download copies or find some other way of getting links between articles (manual scraping, etc.)

**Comment by bksim** [comment]
Definitely. Manual scraping would still rely on urllib right? I don't see a way of getting realtime updates from wikipedia without having this slowdown..maybe we'll just have to settle for downloading the dump and not providing realtime links.
